docs/ref/django-admin.txt

```diff
@@ -632,10 +632,10 @@ If you're in a multi-database setup, you might have fixture data that
 you want to load onto one database, but not onto another. In this
 situation, you can add a database identifier into the names of your fixtures.
 
-For example, if your :setting:`DATABASES` setting has a 'master' database
-defined, name the fixture ``mydata.master.json`` or
-``mydata.master.json.gz`` and the fixture will only be loaded when you
-specify you want to load data into the ``master`` database.
+For example, if your :setting:`DATABASES` setting has a 'users' database
+defined, name the fixture ``mydata.users.json`` or
+``mydata.users.json.gz`` and the fixture will only be loaded when you
+specify you want to load data into the ``users`` database.
```

Comment:
I changed this to use the name "users" as is used for a second database in the `multi-db` topic guide.

The name "master" was even confusing before, since it's not "default" but it sounds like the 'the default database'

Reply:
Good shout.

Reply:
Oh, by the way, I didn't mention `docs/man/django-admin.1` as it gets generated from `docs/ref/django-admin.txt` before release, but you could make this "master" → "users" change there directly if you like. It's straightforward.

Reply:
Updated the man page.